Source/items.cpp

```diff
@@ -3547,7 +3547,7 @@ bool DoOil(Player &player, int cii)
 	case IPL_KNOCKBACK:
 		return _("knocks target back");
 	case IPL_3XDAMVDEM:
-		return _(/*xgettext:no-c-format*/ "+200% damage vs. demons");
+		return fmt::format(fmt::runtime(_(/*xgettext:no-c-format*/ "+{:d}% damage vs. demons")), 200);

```

Comment:
Please derive this from it's source instead of hardcoding it a secound time

Reply:
ok, great idea!

Reply:
Correction to my above comment.  A great idea for another PR.
AJenbo, my PR was meant to change not the hardcoding of the affix itself, but its description only as it relates to the translation files.  Maybe it would be nice to have the values pulled from itemdat.cpp, but I currently do not know how to do this.  I'm trying to follow how it's done for the other affixes, for example item._iPLToHit and I see entries of this in many places including loadsave.cpp.  Unless I'm missing something simple that can allow me to pass those values, it seems that this is not going to be as simple as what I have done.  Does changing the hardcoding for the translations only absolutely require that we remove the hardcoding for the values themselves?  If so I guess what I have done here is not good enough and is destined to wait because I'm simply not able to do what you are asking and highly doubt anybody else will have any interest to do this.

Reply:
Also I don't know why the checks are failing, I was able to build this in VS.